src/fairseq2/recipes/llama/__init__.py

```diff
@@ -18,3 +19,9 @@ def _setup_llama_cli(cli: Cli) -> None:
         handler=ConvertCheckpointCommandHandler(),
         help="convert fairseq2 LLaMA checkpoints to reference checkpoints",
     )
+
+    group.add_command(
```

Comment:
I think this is fine, but would be great if we could also have a way to automatically dump this config.json in LLM finetuning recipes.

Reply:
Agreed!